lib/index.js

```diff
@@ -115,6 +115,10 @@ class CouchbaseService {
 	 * @param {object} options
 	 */
 	insertCallback(key, value, options, callback) { // callback: (error, result)
+		if (options instanceof Function) {
+			callback = options;
+			options = {};
+		}
```

Comment:
Can you add a newline after this bracket to keep some space between this block and the return?